web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -8,142 +8,51 @@
       @close="edit = false"
     />
     <template v-else>
-      <v-container>
+      <v-toolbar class="grey darken-2 white--text">
+        <v-btn
+          icon
+          @click="navigateBack"
+        >
+          <v-icon color="white">
+            mdi-arrow-left
+          </v-icon>
+        </v-btn>
+        <v-toolbar-title>
+          Dandiset Dashboard
+        </v-toolbar-title>
+        <v-spacer />
+        <DandisetSearchField />
+        <v-btn
+          icon
+          @click="detailsPanel = !detailsPanel"
+        >
+          <v-icon color="white">
+            <template v-if="detailsPanel">
+              mdi-chevron-right
+            </template>
+            <template v-else>
+              mdi-chevron-left
+            </template>
+          </v-icon>
+        </v-btn>
+      </v-toolbar>
+      <v-container
+        fluid
+        class="grey lighten-4"
+      >
         <v-row>
+          <v-col>
+            <DandisetData
```

Comment:
It's unclear to me reading this what `DandisetData` is vs. `DandisetDetails`. It looks like `DandisetData` is the main content, and `DandisetDetails` is the collapsible side bar. Perhaps they could be named more clearly?